### DISCUSSION TITLE
How long does it take to totally use AI Search feature

### DISCUSSION THREAD

**Discussion by khanhan2k2eliai** [meilisearch/product#728] [Experimental features]
I see your notifications in Meilisearch Dashboard, I tend to use your AI Search services but it is not fully released. I would like to know that when can I use this services and when it is released can I still use Hugging Face Embedders like I used locally? 
Thanks!

**Comment by macraig** [comment]
Hi @khanhan2k2eliai , we're currently granting access to the feature via waitlist. I've reached out to your email with the next steps to enable it.

> when it is released can I still use HuggingFace Embedders like I used locally?

HuggingFace embedders will be available on Meilisearch Cloud starting tomorrow.
